tap-snapshots/test/lib/commands/config.js.test.cjs

```diff
@@ -7,9 +7,11 @@
 'use strict'
 exports[`test/lib/commands/config.js TAP config list --json > output matches snapshot 1`] = `
 {
-  "prefix": "{LOCALPREFIX}",
-  "userconfig": "{HOME}/.npmrc",
-  "cache": "{NPMDIR}/test/lib/commands/tap-testdir-config-config-list---json-sandbox/cache",
+  "fetch-retries": 0,
```

Comment:
this is ... weird?

Reply:
Config tests were rewritten to not use sandbox anymore. They definitely need a closer look at the snapshots.